src/main/java/net/fabricmc/loader/api/FabricLoader.java

```diff
@@ -59,6 +59,12 @@ static FabricLoader getInstance() {
 	 */
 	Collection<ModContainer> getAllMods();
```

Comment:
Should `getAllMods()` return every single mod, regardless of weather it's fake or real and add a `getRealMods()` method.

Or should we make this exclusively real mods.

Reply:
I feel like it'd be better to have near-total separation between fake and real mods. I might even want to have `isModLoaded` have a counterpart `isFakeModLoaded` instead of the `||` behavior it has in the current state of the PR.

Reply:
I disagree with boundary here, I think mods should be all equal by default. If someone provides a same mod with the same id as a real mod, there should be a good reason, and they should be treated the same.